datalad/local/clean.py

```diff
@@ -133,8 +132,6 @@ def __call__(*,
                  "temporary annex", FILES_PLURAL),
                 (Path(ANNEX_TRANSFER_DIR), "annex-transfer",
                  "annex temporary transfer", DIRS_PLURAL),
-                (gitdir / Path(SEARCH_INDEX_DOTGITDIR), 'search-index',
-                 "metadata search index", FILES_PLURAL),
```

Comment:
I think this was my reason to keep the const.

Reply:
Kept it in the code